src/bin/nydus-image/inspect.rs

```diff
@@ -569,7 +570,7 @@ Blocks:             {blocks}"#,
     Flags:              {flags}"#,
                 version = sb.version(),
                 inodes_count = sb.inodes_count(),
-                flags = sb.flags()
+                flags = RafsSuperFlags::from_bits(sb.flags()).unwrap()
```

Comment:
Is it better to use `RafsSuperFlags::from_bits_truncated()" instead of `unwrap()`?